spring-cloud-skipper-server-core/src/test/resources/repositories/sources/test/ticktock/ticktock-1.0.1/packages/log/templates/log.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: skipper.spring.io/v1
+kind: SpringCloudDeployerApplication
+metadata:
+  name: log
+  type: sink
+spec:
+  resource: maven://org.springframework.cloud.stream.app:log-sink-rabbit
+  resourceMetadata: maven://org.springframework.cloud.stream.app:log-sink-rabbit:jar:metadata:{{version}}
+  version: {{version}}
+  bool: {{bool}}
```

Comment:
I guess this is for testing the marshalling with the custom Resolver?  Mvoing hese to application or deployment properites would be consistent with the expected schema of the yaml file.

Reply:
This also seems to be the only difference between the 1.0.1 and 1.0.0 versions.  It seems a bit heavy to introduce a whole new package for this small difference.